tsdb/engine/tsm1/mmap_windows.go

```diff
@@ -0,0 +1,11 @@
+// portability wrapper for Mmap/Munmap
+// +build windows
+
+package tsm1
+
+import (
+	"golang.org/x/sys/windows"
+)
+
+var Mmap = windows.Mmap
```

Comment:
I hope influxdb can maintain compilation on 32-bit and Windows. It is quite important for IoT applications. It makes really good sense to run with less bits on some small battery powered devices.